tests/trainer/resources/callback/sft_with_callbacks.ipynb

```
@@ -0,0 +1,747 @@
+    "    \"spec\": {\n",
+    "        \"nodeSelector\": {\"kubernetes.io/hostname\": NOTEBOOK_NODE},\n",
+    "        \"volumes\": [{\"name\": \"work\", \"persistentVolumeClaim\": {\"claimName\": PVC_NAME}}],\n",
+    "        \"containers\": [{\n",
+    "            \"name\": \"node\",\n",
+    "            \"volumeMounts\": [{\"name\": \"work\", \"mountPath\": \"/opt/app-root/src\", \"readOnly\": False}],\n",
+    "        }],\n",
+    "    },\n",
+    "}\n",
+    "\n",
+    "_orig_train = KubernetesBackend.train\n",
+    "def _patched_train(self, **kw):\n",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)